.github/actions/junit2jira/action.yaml

```diff
@@ -0,0 +1,46 @@
+name: 'junit2jira'
+description: 'Convert junit failures into jira tickets'
+branding:
+  icon: 'archive'
+  color: 'blue'
+inputs:
+  jira-token:
+    description: 'Token used to authenticate with Jira.'
+    required: true
+  directory:
+    description: 'Directory with Junit XML files.'
+    required: true
+  threshold:
+    description: 'Minimal number of failed issues that will result in reporting a single cumulative Jira issue.'
+    required: false
+    default: '5'
+runs:
+  using: "composite"
+  steps:
+  - name: Download junit2jira
+    shell: bash
+    env:
+      VERSION: "v0.0.6"
+    run: |
+      set -u
+      LOCATION="https://github.com/stackrox/junit2jira/releases/download/$VERSION/junit2jira"
+      # Skip downloading release if downloaded already, e.g. when the action is used multiple times.
+      if [[ ! -e junit2jira ]]; then
+        curl --retry 5 --silent --show-error --fail --location --output junit2jira "$LOCATION"
```

Comment:
Are we reasonably sure `curl` will not leave partial downloads around in case of a failure? Otherwise perhaps check for executability rather than existence, and `rm -f` to prevent clashes?

Reply:
Checking for executability is good idea. 
Clashes will not happen as `curl` will override previous file – no need to `rm -f`